src/v1/cloud-functions.ts

```diff
@@ -240,9 +240,16 @@ export interface HttpsFunction {
 }
 
 /**
- * The Cloud Function type for Blocking triggers.
+ * The Cloud Function type for Auth Blocking triggers.
```

Comment:
I always prefer this is plural "Cloud Functions" vs "function" for a literal function.

Plussed you in to related internal doc.  This is a losing battle, but I mention it anyway :)